.scrutinizer.yml

```diff
@@ -2,6 +2,9 @@ build:
     environment:
         php:
             version: 7.1
+    tests:
+        override:
+            - phpcs-run --standard=phpcs.xml.dist
 
 before_commands:
     - "composer install --no-dev --prefer-source"
```

Comment:
should be removed IMO, as Scrutinizer handles installing with some caching (but should probably be done separately)